app/controllers/application_controller.rb

```diff
@@ -224,10 +217,10 @@ def confirm_two_factor_authenticated(id = nil)
     authenticate_user!(force: true)
     return prompt_to_setup_mfa unless two_factor_enabled?
     return prompt_to_verify_mfa unless user_fully_authenticated?
-    return prompt_to_setup_mfa if service_provider_mfa_policy.
-                                  user_needs_sp_auth_method_setup?
-    return prompt_to_verify_sp_required_mfa if service_provider_mfa_policy.
-                                               user_needs_sp_auth_method_verification?
+    return prompt_to_setup_mfa if service_provider_mfa_policy.user_needs_sp_auth_method_setup?
+    if service_provider_mfa_policy.user_needs_sp_auth_method_verification?
+      return prompt_to_verify_sp_required_mfa
+    end
```

Comment:
So here it switches to an if/end due to the length of the line, but that breaks the pattern of all the other guards.

Reply:
This doesn't drive me crazy - it's nice to have consistent guard formatting but at the same time there is a limit for inline conditionals. Also, it would be easy to miss the period at the end of line 229 (in the old diff) so I actually find the new version to be more readable, personally.